Source/Core/Core/PowerPC/JitArm64/JitArm64_FloatingPoint.cpp

```diff
@@ -26,12 +26,18 @@ void JitArm64::fabsx(UGeckoInstruction inst)
 	fpr.BindToRegister(d, true);
 	ARM64Reg VB = fpr.R(b);
 	ARM64Reg VD = fpr.R(d);
-	ARM64Reg V0 = fpr.GetReg();
-
-	m_float_emit.FABS(EncodeRegToDouble(V0), EncodeRegToDouble(VB));
-	m_float_emit.INS(64, VD, 0, V0, 0);
 
-	fpr.Unlock(V0);
+	if (fpr.IsLower(d))
+	{
+		m_float_emit.FABS(EncodeRegToDouble(VD), EncodeRegToDouble(VB));
+	}
+	else
+	{
+		ARM64Reg V0 = fpr.GetReg();
```

Comment:
Do you think this is worth? We could also just flush the upper part and enforce fpr.R(d) to be in the REG_LOWER_PAIR state.